docs(vscode): Add Cloud to Local feature announcement (#6634)
Azure/LogicAppsUX: apps/vs-code-designer/src/README.md

> Sign up today for your free Azure account and receive 12 months of free popular services, $200 free credit and 25+ always free services 👉 [Start Free](https://azure.microsoft.com/free/open-source).

> ## Azure Logic Apps - Creating Logic Apps Standard Projects from Portal Exports

Reviewer: ```suggestion
## Azure Logic Apps - Create Standard logic app projects from Azure portal export
```

> Starting with version 5.18.7, the Azure Logic Apps (Standard) extension for Visual Studio Code will provide the ability to create new Logic Apps Standard projects from an application exported from the Azure Portal. For more information, see [Export Logic Apps Standard Application to VS Code](https://go.microsoft.com/fwlink/?linkid=2304930).

Reviewer: [Style guide - Visual Studio Code](https://learn.microsoft.com/en-us/product-style-guide-msft-internal/a_z_names_terms/v/visual-studio-code)

```suggestion
Starting with version 5.18.7, the Azure Logic Apps (Standard) extension for Visual Studio Code provides the capability to create Standard logic app projects from a logic app exported from the Azure portal. For more information, see [Export Standard logic apps from Azure portal to Visual Studio Code](https://go.microsoft.com/fwlink/?linkid=2304930).
```

> ## Azure Logic Apps - Introducing the Logic Apps Hybrid Deployment Model (preview)

Reviewer: ```suggestion
## Azure Logic Apps - Introducing the Azure Logic Apps Hybrid deployment model (preview)
```

> Starting with version 4.107.2, the Azure Logic Apps (Standard) extension for Visual Studio Code introduces Logic Apps Hybrid Deployment Model, a new feature that empowers our customers with additional flexibility and control. This new offering allows you to build and deploy workflows that run on customer-managed infrastructure, providing you with the option to run Logic Apps on-premises, in a private cloud, or even in a third-party public cloud. For more information, see [Introducing the Logic Apps Hybrid Deployment Model (Public Preview)](https://go.microsoft.com/fwlink/?linkid=2293544).

Reviewer: Style guide:
- [third-party](https://learn.microsoft.com/product-style-guide-msft-internal/a_z_names_terms/t/third-party)

```suggestion
Starting with version 4.107.2, the Azure Logic Apps (Standard) extension for Visual Studio Code introduces the Azure Logic Apps Hybrid deployment model, a new feature that gives you more flexibility and control over your infrastructure. This offering lets you build and deploy workflows that run on customer-managed infrastructure, providing you with the option to run Standard logic apps on premises, in a private cloud, or even in a non-Microsoft public cloud. For more information, see [Introducing the Logic Apps Hybrid Deployment Model (Public Preview)](https://go.microsoft.com/fwlink/?linkid=2293544).
```
